packages/snaps-utils/src/snaps.ts

```diff
@@ -176,31 +179,52 @@ export class ProgrammaticallyFixableSnapError extends Error {
 }
 
 /**
- * Calculates the Base64-encoded SHA-256 digest of a Snap source code string.
+ * Gets a checksummable manifest by removing the shasum property and reserializing the JSON using a deterministic algorithm.
  *
- * @param sourceCode - The UTF-8 string source code of a Snap.
+ * @param manifest - The manifest itself.
+ * @returns A virtual file containing the checksummable manifest.
+ */
+function getChecksummableManifest(
+  manifest: VirtualFile<SnapManifest>,
+): VirtualFile {
+  const manifestCopy = manifest.clone() as VirtualFile<any>;
```

Comment:
```suggestion
  const manifestCopy = manifest.clone() as VirtualFile<unknown>;
```

Reply:
We know this to be a manifest, we just need it to be cast to `any` so that we can remove a required property without type issues. Using `unknown` doesn't work in this case.